node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+async function copy() {
+  try {
+    const name = process.argv.slice(2);
+    const newFile = name.slice(1);
```

Comment:
This is an interesting and somewhat awkward way to get the command line arguments. You could use use `process.argv[2]` and `process.argv[3]`.